inflect/__init__.py

```diff
@@ -3890,3 +3890,82 @@ def join(
             sep += " "
 
         return f"{sep.join(words[0:-1])}{final_sep}{words[-1]}"
+    
+    
+    #this function takes a string and returns an array with indexes of every first and last digit in the sentence.
+    #example : test_array = find_num_index("In 2019, mark saved $15 thousand, for a total savings of 54 thousand over eight years." )
+    #print(test_array)
+    #output : [3, 6, 21, 22, 57, 58]
+    # 3 and 6 are the indexes of 2 and 9 of "2019"
+    # 21 and 22 are the indexes of 1 and 5 of "15"
+    #and it keeps going
+    
+    def find_num_index(entry_string):
```

Comment:
This "function" is implemented as a method of "engine", but it's not marked as a staticmethod nor does it take "self" as the first parameter. Do you want an instance method or function? How have you invoked this?